_posts/2020-03-20-status-update.md

```diff
@@ -0,0 +1,94 @@
+## Facter 4 Testing
+
+Tested our modules with Facter 4 using pdksync.
+Except for the Apache module, all modules passed Facter 4 testing.
+(apache failed because of a custom fact in https://github.com/gentoo/puppet-portage.git).
+Thanks alot to [Florin Dragos](https://github.com/florindragos), [Bogdan Irimie](https://github.com/IrimieBogdan), [Oana Tanasoiu](https://github.com/oanatmaria), [Sebastian Miclea](https://github.com/sebastian-miclea), [Gheorghe Popescu](https://github.com/gimmyxd), [Andrei Filipovici](https://github.com/Filipovici-Andrei) for all the help on module testing and the fixes.
+
+## Community
+
+As always we have been working with the brilliant Puppet Community to get their changes in to the modules.
+A few highlights this week:
+
+- [puppetlabs-postgresql#1150](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1150): A pleasure working with [olifre](https://github.com/olifre) to get this fix he authored over the line.
+Kudos on the first RSpec test!
+- [puppetlabs-docker#586](https://github.com/puppetlabs/puppetlabs-docker/pull/586): Many thanks to [fe80](https://github.com/fe80) for this fix!
+- [puppetlabs-inifile#387](https://github.com/puppetlabs/puppetlabs-inifile/pull/387): [alexjfisher](https://github.com/alexjfisher) continues his never ending streak of great contributions!
+- [puppetlabs-chocolatey#206](https://github.com/puppetlabs/puppetlabs-chocolatey/pull/206) is a WIP PR put up by [nmaludy](https://github.com/nmaludy) doing the hard and valuable work of adding Bolt tasks to the Chocolatey module!
+
+We’d like to thank the Agent team for their responsiveness in addressing issues that were causing CI test run failures.
+With so many products spanning so many repositories, even with the best foresight from all parties, there can be unforseen knock on effects from a code change.
+When we discovered the root cause of the issues preventing the Puppet 5 agent installing on some older Linux systems, the Agent team were quick off the mark to address our issues.
+Thanks [ciprianbadescu](https://github.com/ciprianbadescu), [lucywyman](https://github.com/lucywyman), beechtom](https://github.com/beechtom), [underscorgan](https://github.com/underscorgan), [e-gris](https://github.com/e-gris), [GabrielNagy](https://github.com/GabrielNagy) and [sarameisburger](https://github.com/sarameisburger)!
+There's been plenty keeping us busy in our CI infrastructure this week: [puppetlabs-ntp#566](https://github.com/puppetlabs/puppetlabs-ntp/pull/566).
+This has been an interesting issue.
+Thanks to [DavidS](https://github.com/DavidS) and [carabasdaniel](https://github.com/carabasdaniel) for their help with [IAC-615](https://tickets.puppetlabs.com/browse/IAC-615).
+Curiosity will likely get the better of us and we’ll want to know why these failures occurred on xenial.
+For now, we’ve bumped the build runner to bionic.
+Check out [the ticket](https://tickets.puppetlabs.com/browse/IAC-615) if you’re interested or have any ideas.
```

Comment:
```suggestion
There's been plenty keeping us busy in our CI infrastructure this week: 
* the Puppet repositories removed unencrypted HTTP access before all tooling upgrades were in place, breaking puppet 5 agent installs in our pipelines.

It was a good lesson how with so many products spanning so many repositories, even with the best foresight from all parties, there can be unforeseen knock on effects from a code change.
When we discovered the root cause of the issues preventing the Puppet 5 agent installing on some older Linux systems, the Agent team were quick off the mark to address our issues.
We’d like to thank the Agent team for their responsiveness in addressing issues that were causing CI test run failures.
Thanks [ciprianbadescu](https://github.com/ciprianbadescu), [lucywyman](https://github.com/lucywyman), beechtom](https://github.com/beechtom), [underscorgan](https://github.com/underscorgan), [e-gris](https://github.com/e-gris), [GabrielNagy](https://github.com/GabrielNagy) and [sarameisburger](https://github.com/sarameisburger)!
This has been an interesting issue.

* the NTP module started failing in travis: [puppetlabs-ntp#566](https://github.com/puppetlabs/puppetlabs-ntp/pull/566).
[DavidS](https://github.com/DavidS) and [carabasdaniel](https://github.com/carabasdaniel) worked on [IAC-615](https://tickets.puppetlabs.com/browse/IAC-615) and tried to analyse it,
but the only thing that helped was bumping the build runner to bionic.
Curiosity will likely get the better of us and we’ll want to know why these failures occurred on xenial.
Check out [the ticket](https://tickets.puppetlabs.com/browse/IAC-615) if you’re interested or have any ideas.
```

Rearranged the sentences a bit to split the two issues into distinct paragraphs. Makes it easier to read and digest. Also removed the thanks for team members. We're just doing our work ;)